crates/bevy_sprite/src/render/mod.rs

```diff
@@ -133,6 +153,15 @@ impl SpecializedPipeline for SpritePipeline {
             shader_defs.push("COLORED".to_string());
         }
 
+        if key.contains(SpritePipelineKey::HDR) {
+            shader_defs.push("TONEMAPPING_IN_SPRITE_SHADER".to_string());
+        }
+
+        let format = match key.contains(SpritePipelineKey::HDR) {
+            true => ViewTarget::TEXTURE_FORMAT_HDR,
+            false => TextureFormat::bevy_default(),
+        };
```

Comment:
There was a bug here I think. This was supposed to be `!key.contains(...HDR)`
```suggestion
        let mut format = ViewTarget::TEXTURE_FORMAT_HDR;
        if !key.contains(SpritePipelineKey::HDR) {
            shader_defs.push("TONEMAPPING_IN_SPRITE_SHADER".to_string());
            format = TextureFormat::bevy_default();
        }
```